### ISSUE TITLE
Show progress of the selected files

### ISSUE THREAD

**Issue by bravechamp** [transmission/transmission#5942]
### What is the issue?

Transmission should show progress of the selected files by default. In many cases, I prefer to not download a portion of the torrent. Seeing the progress compared to the whole torrent doesn't reveal much about the real progress of the download.

Here's an extreme example of this:
![Screenshot](https://github.com/transmission/transmission/assets/48980452/da40eb94-a197-4694-91a5-5402b2f753a1)

### Which application of Transmission?

macOS app

### Which version of Transmission?

4.0.3 (6b0e49bbb2)

**Comment by killemov** [comment]
Yes. The data is there, just not rendered. [Shift](https://github.com/killemov/Shift) is supposed to show everything defined in the RPC and this has been in there from day one.
